packages/eslint-plugin-next-on-pages/src/rules/no-unsupported-configs.ts

```diff
@@ -93,7 +93,7 @@ const ruleSchema = {
 const rule: Rule.RuleModule = {
 	create: context => {
 		const code = context.sourceCode;
-		const exportedConfigName = context.filename.match(/next\.config\.js$/)
+		const exportedConfigName = context.filename.match(/next\.config\.[mc]?js$/)
```

Comment:
I actually don't think that Next picks up `next.config.cjs` files, I can't see that in their documentation and when I tried to use it locally it wasn't indeed getting picked up, I think they only support `next.config.js` and `next.config.mjs`

so could remove the `c` here and remove the `cjs` extension mentioned in the rest of this PR? :pray: 

Unless I'm mistaken, please let me know if I am :slightly_smiling_face:

Reply:
You are right, I only assumed, never looked it up. But it doesn't really make sense to have a .cjs config file when ESM is supported.

Reply:
Probably.